api_tests/base/test_filters.py

```diff
@@ -222,3 +229,51 @@ def test_convert_value_float(self):
         field = FakeSerializer._declared_fields['float_field']
         value = self.view.convert_value(value, field)
         assert_equal(value, 42.0)
+
+
+
+class TestODMOrderingFilter(ApiTestCase):
+    def test_filter_queryset_fails(self):
+        class query:
+            title = ' '
+            def __init__(self, title):
+                self.title = title
+            def __str__(self):
+                return self.title
+        query_s = []
+        q1 = query('NewProj')
+        query_s.append((q1))
```

Comment:
What's up with the extra parens here? Let's drop them here and throughout these two new tests.